src/index.ts

```diff
@@ -168,9 +180,10 @@ export class Paginator {
    * @param {string} methodName - Name of the method to streamify.
    * @return {function} - Wrapped function.
    */
-  streamify(methodName: string) {
-    // tslint:disable-next-line:no-any
-    return function(this: {[index: string]: Function}, ...args: any[]) {
+  streamify<T = {}>(methodName: string) {
```

Comment:
Using `<T = {}>` will probably introduce a lot of narsty compilation errors for folks.  Can we use `<T = any>` instead?

Reply:
Sure!